### DISCUSSION TITLE
Window95 and real Hardware. Connections to COM, NIC, PCMCIA and Parallel ports

### DISCUSSION THREAD

**Discussion by nt4boy** [86Box/86Box#4801] [Q&A]
I've imaged a lap top of the period, and have it working well in 86Box. This machine is used to communicate with a car data logger and other electronic devices on the chassis. To my surprise, after setting this up on a W11 laptop and supplying it with USB COM ports and USB NIC (via NPcap)  I can upload and download from these devices via COM and NIC. Top marks to the 86Box team.
However, on another similar old (win95) laptop and some of the comms are via a PCMCIA card which I think carries a licence key, but is also passing through data. (I have tried this card in a W7 laptop, without the W95 software, just to see what the OS makes of the card. I was expecting it to be an NIC or COM port as well, but although I can see a hardware ID, its evidently bespoke no PCI/VEN codes etc.) 
Connecting up the PCMCIA card to the new laptop will interesting. USB adapters are only good for memory cards as far as I can tell, so I am rounding up hardware that will bridge from the Card in a PCI slot to a PCIe adaptor and then hopefully via Thunderbolt port.
Anyway, the question is, Assuming I get this card visible to the W11 OS,  could there be a way for my Win95 86Box VM to talk to it?
I've yet another Win95 laptop that has a variation on this theme by having a parallel port dongle. 
Obviously these old laptops are rare, and hard to keep alive, but the hardware they talk to lives on, and the software won't be updated to anything that will run on modern machines, so 86Box does appear to be a saviour.
I'd appreciate any pointers and thoughts.

**Comment by Torinde** [comment]
PCI Passthrough:
- https://github.com/86Box/86Box/discussions/4759#discussioncomment-15046236
